Source/JavaScriptCore/jit/GCAwareJITStubRoutine.h

```diff
@@ -116,6 +123,8 @@ class PolymorphicAccessJITStubRoutine : public GCAwareJITStubRoutine {
     FixedVector<RefPtr<AccessCase>> m_cases;
     FixedVector<StructureID> m_weakStructures;
     FixedVector<Identifier> m_identifiers;
+    Ref<WatchpointSet> m_watchpointSet;
```

Comment:
Maybe we should have a EmbeddedWatchpointSet which is just `WatchpointSet` without the `ThreadSafeRefCounted<WatchpointSet>`?

Reply:
Yeah, I think this is good extension in the later patch. For now, I'm just using the existing mechanism.